python/ray/_private/runtime_env/cgroup.py

```diff
@@ -0,0 +1,19 @@
+import logging
```

Comment:
remove this file and submit it as part of the PR for plugin?

Reply:
Thanks for your suggestions, I have deleted it.